packages/scripts/README.md

```
@@ -0,0 +1,9 @@
+# flyonui-scrips
+
+Useful scripts to simplify the development workflow:
+
+1) `yarn generate:resolutions`: It generates the right resolutions for the same packages being used in different monorepo.
+   packages.
+2) `generate:components-api`: It generates the api of the whole components, to be shown on the docs and exported from the
+   `core` package into [ComponentsApi.json](https://github.com/michaelcozzolino/flyonui-vue/tree/2.x/packages/docs/Api/Lib/ComponentsApi.json).
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)